cif_pow.dic

```diff
@@ -8682,6 +8920,25 @@ save_pd_spec.description
 
 save_
 
+save_pd_spec.diffractogram_id
+
+    _definition.id                '_pd_spec.diffractogram_id'
+    _definition.update            2023-01-12
+    _description.text
+;
+    The diffractogram (see _pd_diffractogram.id) to which the specimen-
+    preparation details relate.
+;
+    _name.category_id             pd_spec
+    _name.object_id               diffractogram_id
+    _name.linked_item_id          '_pd_diffractogram.id'
+    _type.purpose                 Link
+    _type.source                  Related
+    _type.container               Single
+    _type.contents                Text
+
+save_
+
```

Comment:
`PD_SPEC` is an interesting one, because it says it is about the specimen but really it is mostly about how the specimen was prepared and mounted for a specific measurement.  Therefore it does make sense that there is a pointer to diffractogram.id, being the particular measurement that these preparation details relate to.

**But** we might prepare a specimen once and collect 100 diffraction patterns at different temperatures, in which case you would expect 100 lines in `PD_SPEC`, all identical except for `diffractogram_id`. So there must be a better way.

Probably better therefore is to switch, so that a diffractogram points to a `pd_spec.id`. That way multiple measurements at differing environmental conditions would simply point to the same `pd_spec.id`.

Reply:
switched